nl.sarine.gpx-viewer.json

```diff
@@ -0,0 +1,87 @@
+{
+    "app-id" : "nl.sarine.gpx-viewer",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gpx-viewer",
+    "rename-desktop-file" : "gpx-viewer.desktop",
+    "rename-appdata-file" : "gpx-viewer.metainfo.xml",
+    "rename-icon" : "gpx-viewer",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--env=G_ENABLE_DIAGNOSTICS=1"
```

Comment:
what does that do? users are unlikely to see anything written on the console.

Reply:
Comes from gnome maps. To show deprecations I guess